ref: `remote modify` `gdrive_acknowledge_abuse` and `profile` (#3795)
iterative/dvc.org: content/docs/user-guide/privacy.md

DVC stores your credentials on your machine, for later reuse by you. **Use
caution when using Google Drive DVC remotes on shared machines.**

> By default, OAuth tokens are cached in a global directory (e.g.
> `~/.cache/pydrive2fs` on linux). See
> [`gdrive_user_credentials_file`](https://dvc.org/doc/command-reference/remote/modify#google-drive)
> for more info.

Reviewer: ```suggestion
By default, OAuth tokens are cached in a global location (e.g.
`~/.cache/pydrive2fs` on Linux, see [details]).

[details]: https://dvc.org/doc/command-reference/remote/modify#google-drive
```
